sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2023-2024.bpmn

```diff
@@ -437,7 +437,7 @@
       <bpmn:extensionElements>
         <zeebe:ioMapping>
           <zeebe:output source="=  if (calculatedDataTotalFamilyIncome &#60;= dmnPartTimeAwardFamilySizeVariables.limitAwardBCAGIncomeCap)&#10;then dmnPartTimeAwardAllowableLimits.limitAwardBCAGAmount&#10;else&#10;  max(dmnPartTimeAwardAllowableLimits.limitAwardBCAGAmount - ((calculatedDataTotalFamilyIncome - dmnPartTimeAwardFamilySizeVariables.limitAwardBCAGIncomeCap) * dmnPartTimeAwardFamilySizeVariables.limitAwardBCAGSlope),&#10;     100&#10;  )" target="federalAwardBCAGAmount" />
-          <zeebe:output source="=if (awardEligibilityBCAG = true)&#10;then&#10;  if (calculatedDataTotalRemainingNeed3 &#62; federalAwardBCAGAmount)&#10;  then federalAwardBCAGAmount&#10;  else calculatedDataTotalRemainingNeed3&#10;else&#10;0" target="provincialAwardNetBCAGAmount" />
+          <zeebe:output source="=if ((awardEligibilityBCAG = true) and (federalAwardBCAGAmount &#62;= 100))&#10;then&#10;  min(&#10;     calculatedDataTotalRemainingNeed3,&#10;     min(dmnPartTimeAwardAllowableLimits.limitAwardBCAGAmount, federalAwardBCAGAmount )&#10;  )&#10;else&#10;0" target="provincialAwardNetBCAGAmount" />
```

Comment:
Minor but there is an unnecessary white space before the parenthesis in `federalAwardBCAGAmount )`.

Reply:
The extra white space before the parenthesis is removed in my latest commit. Please have a look.